src/execute.ts

```diff
@@ -66,6 +67,7 @@ function getDatabaseRef({
   }
 
   let ref: any
+  let resolver: any = getDocs
```

Comment:
Thanks for the fix @steelbrain! Very clever.

Is there any way we can type `ref`/`resolver`? Think it would've help prevent the bug.

Reply:
Firestore typings are too hairy for my patience :)

Feel free to take a stab at it

Reply:
https://github.com/steelbrain/apollo-link-firestore/pull/7 :)